src/blog/db/config.py

```diff
@@ -18,7 +18,7 @@ class Settings(BaseSettings):
     CLIENT_ORIGIN: str
 
     class Config:
-        env_file = '../.env'
+        env_file = '.env'
```

Comment:
A note for future: 
it's more secure to use environment variables (like when you just export some value into a variable) in comparison to storing them in dotenv file.

Reply:
Alright then, maybe I should get rid of .env file.